library/Requests.php

```diff
@@ -187,8 +187,7 @@ protected static function get_transport($capabilities = array()) {
 
 		// Don't search for a transport if it's already been done for these $capabilities
 		if (isset(self::$transport[$cap_string]) && self::$transport[$cap_string] !== null) {
-			$class = self::$transport[$cap_string];
-			return new $class();
+			return self::$transport[$cap_string];
 		}
 		// @codeCoverageIgnoreEnd
 
```

Comment:
👇🏻 I'm noticing that the double listing of the "default" transports here and in the above `add_transport()` method is bugging me. Will add a fix for this to the "magic constants" set of commits.